packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration-modal.tsx

```diff
@@ -18,7 +18,7 @@ const MultipliersExpirationModal = observer(({ is_open, toggleModal }: TMultipli
             should_header_stick_body={false}
             height='auto'
             width='calc(100vw - 3.2rem)'
-            title={localize('Expiration')}
+            title={<Localize i18n_default_text='Expiration' />}
```

Comment:
Could you please delete `localize` from import if it's not used in this file anymore?

Reply:
@kate-deriv it's used for Button text.